api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -90,6 +91,8 @@ public class AdministrationServiceImpl extends BaseOpenmrsService implements Adm
 	@Autowired
 	@Qualifier("openmrsEventListeners")
 	private EventListeners eventListeners;
+
+	private static final String SERIALISER_WHITE_LIST_TYPES = "serializerWhiteListTypes";
```

Comment:
What is the use of declaring this in a constant that is private and used only once?